is-subdir/index.js

```diff
@@ -2,8 +2,16 @@
 const betterPathResolve = require('better-path-resolve')
 const path = require('path')
 
-module.exports = function isSubdir (parentDir, subdir) {
+function isSubdir (parentDir, subdir) {
   const rParent = `${betterPathResolve(parentDir)}${path.sep}`
   const rDir = `${betterPathResolve(subdir)}${path.sep}`
   return rDir.startsWith(rParent)
 }
+
+isSubdir.strict = function isSubdirStrict (parentDir, subdir) {
+  const rParent = `${betterPathResolve(parentDir)}${path.sep}`
+  const rDir = `${betterPathResolve(subdir)}${path.sep}`
+  return rDir !== rParent && rDir.startsWith(rParent)
+}
```

Comment:
I wonder how will this work when the paths have different cases and the file system is case insensitive.

`C:\foo` and `C:\FOO` is the same on Windows.

Reply:
I added the dumb `!==` comparison, but `.startsWith()` isn't case sensitivity aware either, so I didn't read that requirement from the code. Maybe `betterPathResolve()` is better enough to normalize case for us?